### ISSUE TITLE
Can't get it to work with HTTPS on 443

### ISSUE THREAD

**Issue by Puneeth-n** [TheDeveloper/http-aws-es#30]
**version: 2.0.3**
```
npm run test -- --endpoint https://my-aws-es.eu-west-1.es.amazonaws.com --region eu-west-1
```

ends with socket error however,
```
npm run test -- --endpoint http://my-aws-es.eu-west-1.es.amazonaws.com --region eu-west-1
npm run test -- --endpoint https://my-aws-es.eu-west-1.es.amazonaws.com:80 --region eu-west-1
```
 works fine

**Comment by kagonzalez84** [comment]
Same happens to me

**Comment by devraz** [comment]
Same here.

**Comment by ef4** [comment]
I think I have a fix in #33, please give it a whirl.

**Comment by Puneeth-n** [comment]
@ef4 Thanks! works!

**Comment by devraz** [comment]
@ef4 The fix in the branch fix-30 works for me too. 

When can we expect this to be published? Thanks

**Comment by rmm5t** [comment]
@ef4 Thanks! Your branch fixed our AWS deployment after attempting to upgrade http-aws-es.

Considering AWS Elasticsearch defaults endpoint URLs to `https`, this bug seems pretty critical, warranting an immediate merge and release of a new v2.0.4 package, no?

**Comment by TheDeveloper** [comment]
Merged in 2.0.4.

Thanks all
